foosballv2s/Source/Activities/AuthActivity.cs

```diff
@@ -46,6 +52,58 @@ protected override void OnCreate(Bundle savedInstanceState)
             tabLayout.SetupWithViewPager(viewPager);
         }
 
+        [Export("ChangeLanguage")]
+        public async void ChangeLanguage(View view)
+        {
+            dialog = new Dialog(this);
+            dialog.SetContentView(Resource.Layout.language_spinner);
+            dialog.SetTitle(Resource.String.choose_lang);
+            dialog.SetCancelable(false);
+            dialog.Show();
+
+            languages = (RadioGroup)dialog.FindViewById(Resource.Id.languages);
```

Comment:
Maybe I do not understand something here, but do you access the same dialog here that is used above?